modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-azure-user-infra.adoc
+
+[id="installation-azure-user-infra-uploading-rhcos_{context}"]
+= Uploading the {op-system} cluster image and bootstrap ignition config file
+
+The Azure client does not support deployments based on files existing locally;
+therefore, you must copy and store the {op-system} virtual hard disk (VHD)
+cluster image and bootstrap ignition config file in a storage container so they
+are accessible during deployment.
+
+.Prerequisites
+
+* Configure an Azure account.
+
+* Generate the Ignition config files for your cluster.
+
+.Procedure
+
+. Create an Azure storage account to store the VHD cluster image:
++
+----
+$ az storage account create -g ${RESOURCE_GROUP} --location ${AZURE_REGION} --name ${CLUSTER_NAME}sa --kind Storage --sku Standard_LRS
+----
++
+[WARNING]
+====
+The Azure storage account name must be between 3 and 24 characters in length and
+use numbers and lower-case letters only. If your `CLUSTER_NAME` variable does
+not follow these restrictions, you must manually define the Azure storage
+account name. For more information on Azure storage account name restrictions,
+see link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/error-storage-account-name[Resolve errors for storage account names]
+in the Azure documentation.
+====
+
+. Export the storage account key as an environment variable:
++
+----
+$ export ACCOUNT_KEY=`az storage account keys list -g ${RESOURCE_GROUP} --account-name ${CLUSTER_NAME}sa --query "[0].value" -o tsv`
+----
+
+. Choose the {op-system} version to use and export the URL of its VHD to an
+environment variable:
++
+----
+$ export VHD_URL=`curl -s https://raw.githubusercontent.com/openshift/installer/release-4.4/data/data/rhcos.json | jq -r .azure.url`
```

Comment:
Hello @codyhoag , I tested last week, but I noticed today that I did not use the same RHCOS image that you have in this PR. My understanding is that you would need to use a 4.3 RHOS image with 4.3 so I used `export VHD_URL=`curl -s https://raw.githubusercontent.com/openshift/installer/release-4.3/data/data/rhcos.json | jq -r .azure.url`

Reply:
@e-tienne good catch! I just updated the RHCOS image to 4.3